src/Cli/Microsoft.DotNet.Cli.Utils/UILanguageOverride.cs

```diff
@@ -4,11 +4,11 @@
 using System;
 using System.Globalization;
 
-namespace Microsoft.DotNet.Cli
+namespace Microsoft.DotNet.Cli.Utils
```

Comment:
I moved this file to Utils because a) much of the shared utility logic for the CLI was already in this project and b) this project already had internals visible to the test framework, where I wanted to add a `WithCulture` extension method.